### ISSUE TITLE
Training loss is not decreasing

### ISSUE THREAD

**Issue by SamratSinghRathore** [mberkay0/pretrained-backbones-unet#5]
Hello, I am using Unet decoder and HRNet encoder model but i realize that the training loss is not decreasing, instead it seems to remain the same. Also the training logs appear in separate lines which is a problem becuase 1 epoch take around 200 lines and not gets output in one line. 
![Screenshot from 2023-06-16 19-27-31](https://github.com/mberkay0/pretrained-backbones-unet/assets/99594852/ed3af484-8c65-4827-bb6c-8813812bc746)

I have trained it for 10 epochs but the loss keeps jumping between 0.99 and 1

**Comment by mberkay0** [comment]
Greetings @SamratSinghRathore, 
The divergence/non-convergence of a model may depend on more than one situation. I recommend trying different parameters and different encoders. You may also consider freezing specific layers of the encoder. Another option is to freeze the entire encoder and train only the decoder. 
In addition, please ensure to pre-process your data according to the backbone network model you have chosen.
Best regards.

**Note:** The separate line verbose is due to the 'tqdm' library. This problem may occur in the Jupyter Notebook. I will update the 'tqdm.auto' module.
Perhaps you can use this for a quick solution:

```python

from tqdm import tqdm

tqdm._instances.clear()
```

**Comment by SamratSinghRathore** [comment]
Hello,

Thank you for the reply. Will implement and get back to you.

Regards
Samrat

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
